src/main/flight/pid.c

```diff
@@ -590,6 +637,13 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
         }
 #endif // USE_YAW_SPIN_RECOVERY
 
+        // mix in a correction for accrued attitude error
+        float acCorrection = 0;
```

Comment:
`acCorrection` is inelegant - mixing it in and out of currentPidSetpoint repeatedly. Maybe it can be 'decoupled' a bit?

Reply:
Yeah originally I had stickPidSetpoint and currentPidSetpoint. Initially seems I could get it done easier without the stickPidSetpoint. But I agree it's not looking too hot in the end.

Reply:
Ok I've decoupled a bit.